.travis.yml

```diff
@@ -28,8 +28,15 @@ rvm:
   - 2.3.0
   - ruby-head
 matrix:
+  include:
+    - rvm: jruby-9.0.4.0 # Latest compiled version in http://rubies.travis-ci.org
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='-Xcompat.version=2.0 --server -Xcompile.invokedynamic=false'"
```

Comment:
Thanks @kares No wonder I couldn't find docs for it anymore or much in the code! https://twitter.com/hazula/status/705225976926572546